Rhino sdk validating dir dd (#64)
RhinoHealth/user-resources: examples/rhino-sdk/cox/README.md

> ## This folder contains a minimal working example for computing Cox proportional hazard metrics using the Rhino SDK.

Reviewer: ```suggestion
## This folder contains a working example for computing Cox Proportional Hazards (Cox PH) using the Rhino SDK.
```

## Files

> - `cox_sampledata1.csv` – sample dataset 1
> - `cox_sampledata2.csv` – sample dataset 2

Reviewer: ```suggestion
- `cox_sample_data1.csv` – sample dataset 1
- `cox_sample_data2.csv` – sample dataset 2
```

- `cox.ipynb` – interactive notebook demonstrating the end-to-end process

## How to Use

> 1. Upload both CSV files as datasets in your Rhino project.

Reviewer: ```suggestion
1. Import both CSV files as datasets in your Rhino project.
```

2. Run the cells in `cox.ipynb` to compute and visualize the Cox regression results.

> 3. All column names are pre-formatted and aligned with the metric's expected input schema.

Reviewer: This isn't an instruction
```suggestion
```
